### DISCUSSION TITLE
Autorisierung für SELECT/Endpoint_ID@Query_ID@View_ID

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/datalinq-community#17] [General]
Bei der Bearbeitung von einer nicht räumlichen Tabelle mit DataLinq werden über JS ../select/Endpoint_ID@UpdateQuery_ID alle Werten des Datensatz als REST-Parameter übergeben.  
Dann kommt die Fehlermeldung:
![image](https://github.com/gis-eni/datalinq-forum/assets/1842500/1ff9f51e-dc39-40c5-a460-156254e8c66c)
Der Endpoint und die Query sind nicht frei zugängig.

Schon bei der View bekommt man diese Fehlermeldung, wenn man REPORT durch SELECT ersetzt.

In der Entwicklungsumgebung sind die Berechtigungen gleich gesetzt, aber da kann man REPORT durch SELECT ersetzten und man bekommt keine Fehlermeldung.

In der Entwicklungsumgebung funktioniert der JS-Aufruf ../select/Endpoint_ID@UpdateQuery_ID mit der Übergabe der Werte die zu aktualisieren sind. Aber nur wenn aus der Datalinq-Entwicklung gestartet wird - sprich mit dem DL-Admin-Account?

Kann man den Select-Aufruf nur als Admin ausführen, oder wenn keine Zugriffsbeschränkung drauf ist?

**Comment by gis-eni** [comment]
Nein, ein Select funktioniert für jeden berechtigten Benutzer
